lib/stale.js

```diff
@@ -198,8 +198,27 @@ module.exports = class Stale {
   }
 
   since (days) {
-    const ttl = days * 24 * 60 * 60 * 1000
-    let date = new Date(new Date() - ttl)
+    const daysInMilliseconds = 24 * 60 * 60 * 1000
+    const now = Date.now()
+    let daysToCount = 0
+    let daysAgo = 0
+
+    const { skipWeekends } = this.config
+    if (skipWeekends) {
+      while (daysToCount < days) {
+        daysAgo++
+        const dayOfWeek = new Date(now - daysAgo * daysInMilliseconds).getDay()
```

Comment:
Saying `(now - daysAgo * daysInMilliseconds)` is the equivalent of saying

```
console.log(7 - 4 * 5); // 7 - 20
// expected output: -13
```

Maybe you want: `const dayOfWeek = new Date((now - daysAgo) * daysInMilliseconds).getDay()`

Reply:
This is correct. `now` is already a timestamp in milliseconds, so `daysAgo * daysInMilliseconds` is converting to the same unit. If anything, it might be worth adding parens around the multiplication or assigning another variable for clarity, or renaming `daysInMilliseconds` to `millisecondsPerDay` or similar? (That said, the improvement @bkeepers suggested might make this code unnecessary, so probably good to do that first.)